mlir/lib/Dialect/OpenMP/IR/OpenMPDialect.cpp

```diff
@@ -2144,6 +2144,12 @@ void LoopNestOp::gatherWrappers(
     wrappers.push_back(wrapper);
     parent = parent->getParentOp();
   }
+
+  // omp.parallel can be misidentified as a loop wrapper when it's not taking
+  // that role but it contains no other operations in its region (e.g. parallel
+  // do/for).
+  if (llvm::isa<omp::ParallelOp>(wrappers.back()))
```

Comment:
`isa` is one of the definitions that is also [imported into the mlir namespace](https://github.com/llvm/llvm-project/blob/bf76290de48512f59f62eff20d28135c3f918ea5/mlir/include/mlir/Support/LLVM.h#L106)
```suggestion
  if (isa<omp::ParallelOp>(wrappers.back()))
```